oe_corporate_blocks.post_update.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Corporate blocks Demo post updates.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Delete the config and menu for Site Switcher.
+ */
+function oe_corporate_blocks_post_update_00001_delete_all_of_site_switcher() {
+  // Delete the config.
+  \Drupal::configFactory()->getEditable('oe_corporate_blocks.data.site_switcher')
+    ->delete();
+
+  // Delete the menu.
+  \Drupal::entityTypeManager()->getStorage('menu_link_content')
```

Comment:
When running the update I get 'The "menu_link_content" entity type does not exist.'

Reply:
Good catch.
I remove the code to remove the link because a 'drush cr' rebuilds the menu (and remove automatically the old link menu)